src/routes.js

```diff
@@ -3,11 +3,13 @@ import SignUp from './views/SignUp';
 import SignIn from './views/SignIn';
 import Yahtzee from './components/Yahtzee';
 import Nertz from './views/games/Nertz';
```

Comment:
It's still there?

Reply:
New error is the empty <template></template> tag in your vue file

Reply:
saw that, added a div in there should fix it

Reply:
I had some files not tracking.